Makefile

```diff
@@ -17,9 +17,9 @@ help:
 
 DOCKER_NETWORK_NAME=reviewer_build
 
-TRAVIS_COMMIT ?= "local"
+IMAGE_TAG ?= "local"
```

Comment:
What does `?=` do?

Reply:
If the environment variable isn't already defined, it's defined as `local`

Reply:
See page 62: https://www.gnu.org/software/make/manual/make.pdf